session/session.go

```diff
@@ -77,6 +97,9 @@ func (s *ZKSession) Subscribe(subscription chan<- ZKSessionEvent) {
 func (s *ZKSession) notifySubscribers(event ZKSessionEvent) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
+	if s.log != nil && event != SessionClosed {
```

Comment:
Why is there an `event != SessionClosed`? Why does subscribers not want that event?